modules/labs-react/common/stories/Box.stories.mdx

```diff
@@ -0,0 +1,92 @@
+import {Meta} from '@storybook/addon-docs/blocks';
+
+import {Box} from '@workday/canvas-kit-labs-react/common';
+// examples
+import {Border} from './examples/Border';
+import {Color} from './examples/Color';
+import {Depth} from './examples/Depth';
+import {Layout} from './examples/Layout';
+import {Position} from './examples/Position';
+import {Space} from './examples/Space';
+
+<Meta title="Labs/Common/Box" component={Box} />
+
+# Box
+
+[![LABS: Beta](https://img.shields.io/badge/LABS-beta-orange)](https://github.com/Workday/canvas-kit/tree/master/modules/labs-react/README.md)
+This component is work in progress and currently in pre-release.
+
+`Box` is a primitive component that provides a common, ergonomic API around Canvas design tokens. It
+is highly flexible, and its primary purpose is to build other components.
+
+## Style Props
+
+While `Box` supports `as`, `ref`, and all semantic element props, style props are at its heart.
+Style props are special design-system-aware props that allow you to use Canvas design tokens
+efficiently as you build components. Style props are better understood by their categories, which
+are listed below. Each category has a more detailed explanation in the following subsections.
+
+### Categories
+
+- [Border](#border)
+- [Color](#color)
+- [Depth](#depth)
+- [Layout](#layout)
+- [Position](#position)
+- [Space](#space)
+
+### Border
+
+Border style props allow you to adjust the border styles of components. Some props, such as
+`borderRadius` and `borderColor`, are connected to our design tokens. This makes it really simple to
+use Canvas values as you build UI.
+
+#### Example
+
+<ExampleCodeBlock code={Border} />
+
+### Color
+
+Color style props allow you to adjust the color and background color of components. There are three
+color props available: `color`, `backgroundColor`, and `bgColor` (an alias for `backgroundColor`).
+
+#### Example
+
+<ExampleCodeBlock code={Color} />
+
+### Depth
+
+Depth style props allow you to adjust the visual elevation of a component using our depth tokens.
+`Box` applies these tokens via the `depth` prop.
+
+#### Example
+
+<ExampleCodeBlock code={Depth} />
+
+### Layout
+
+Layout style props allow you to adjust the display, dimensions, and overflow of a component. These
+props are not connected to our design tokens, but they are often useful for building components.
+
+#### Example
+
+<ExampleCodeBlock code={Layout} />
+
+### Position
+
+Position style props allow you to adjust the position of a component. These props are not connected
+to our design tokens, but they are often useful for building components.
+
+#### Example
+
+<ExampleCodeBlock code={Position} />
```

Comment:
Nit: this example would be a bit clearer if the container around the 3 boxes were taller. That way, it would be more obvious the "Center" in `Center-Left Position`, `Center Position`, and `Center-Right Position` refers to vertical centering. Otherwise, I'd just use `Left`, `Center`, and `Right`.